### PR TITLE
fix: fix style-components write style and composeWithDevTools use

### PR DESCRIPTION
**Pull request by MrZhang123**
when i read the code i find style-components write not good in App.js , I think maybe forget add semicolon 😄, and other questions is why use `composeWithDevTools` in productions ? so I try to fixed it . Let me know what you think.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ import { configureStore } from './root.store';
 
 const Container = styled.View`
   align-items: center;
-  background-color: ${colors.white}
+  background-color: ${colors.white};
```
Comment (chinesedfan) [inline App.js]: Nice catch! Our stylelint script only checks `src/**/*.js`.

File: root.store.js [hunk 2]
```diff
@@ -31,14 +31,11 @@ let store;
...
-    rootReducer,
-    compose(getMiddleware(), ...getEnhancers())
-  );
-} else {
-  store = createStore(
     rootReducer,
     composeWithDevTools(getMiddleware(), ...getEnhancers())
   );
+} else {
+  store = createStore(rootReducer, compose(getMiddleware(), ...getEnhancers()));
```
Comment (chinesedfan) [inline root.store.js]: > Reactotron is different, it differs from React Native Debugger, and both of these applications do not work together, so composeWithDevTools is not added when Reactotron is enabled.

According to #330, maybe we should fix as,
```
if (__DEV) {
    if (process.env.TRON_ENABLED) {
        // Reactotron.createStore
    } else {
        // composeWithDevTools
    }
} else {
    // createStore
}
```

File: src/auth/screens/auth-profile.screen.js [hunk 3]
```diff
@@ -42,6 +43,8 @@ const mapDispatchToProps = dispatch =>
     dispatch
   );
 
+const hairlineWidth = StyleSheet.hairlineWidth;
```
Comment (chinesedfan) [inline src/auth/screens/auth-profile.screen.js]: Hi @MrZhang123, maybe you should create another PR to change all borders to hair line.

File: src/auth/screens/auth-profile.screen.js [hunk 4]
```diff
@@ -145,6 +149,7 @@ class AuthProfile extends Component {
 
           {hasInitialUser &&
             user.bio &&
+            !isPending &&
```
Comment (chinesedfan) [inline src/auth/screens/auth-profile.screen.js]: If it matters, we need a new PR, too.

File: src/components/section-list.component.js [hunk 5]
```diff
@@ -15,9 +15,9 @@ type Props = {
   noItemsMessage: string,
   buttonAction: Function,
 };
-
+const hairlineWidth = StyleSheet.hairlineWidth;
 const Section = styled.View`
-  margin-top: 15;
+  /* margin-top: 15; */
```
Comment (chinesedfan) [inline src/components/section-list.component.js]: Do you want to remove this line?

File: src/components/section-list.component.js [hunk 6]
```diff
@@ -35,8 +35,8 @@ const TitleView = styled.View`
 const StyledList = styled(List).attrs({
   containerStyle: {
     marginTop: 0,
-    borderBottomColor: colors.grey,
-    borderBottomWidth: 1,
+    borderBottomColor: colors.greyLight,
```
Comment (chinesedfan) [inline src/components/section-list.component.js]: Color is changed.

File: src/components/section-list.component.js (hunk 5 above)
Comment (MrZhang123) [reply src/components/section-list.component.js]: yes, i think some style need to change, so i change something , but it's no complete

Comment (chinesedfan) [reply src/components/section-list.component.js]: Sure, friendly reminding, please keep a PR independent. And you can test different kinds of styles locally or in other branches. Then our maintainers will not be notified to review. :)

**Comment by chinesedfan** [comment]
Landed in https://github.com/gitpoint/git-point/commit/738ab3435a8f88f7813e847a7efb7bd0075cde77. And thank @MrZhang123 all the same.

**Pull request closed without merging**